drivers/hih6130/hih6130.c

```diff
@@ -84,7 +84,7 @@ static inline int hih6130_get_humidity_temperature_raw(const hih6130_t *dev, uin
 
     i2c_acquire(dev->i2c);
 
-    if (i2c_read_bytes(dev->i2c, dev->addr, &buf[0], sizeof(buf)) != sizeof(buf)) {
+    if (i2c_read_bytes(dev->i2c, dev->addr, &buf[0], sizeof(buf), 0) != sizeof(buf)) {
```

Comment:
should `!= 0`, does not return read bytes anymore

Reply:
Oops, will fix